api/types/appserver.go

```diff
@@ -185,15 +190,17 @@ func (s *AppServerV3) GetTunnelType() TunnelType {
 	switch {
 	case s.Origin() == OriginOkta:
 		return OktaTunnel
+	case s.GetSubKind() == SubKindOktaApp:
+		return OktaTunnel
```

Comment:
@mdwn Do we really need subkind though, in addition to "origin: okta"? I would keep things simple, if we don't introduce subkind we wouldn't need those caching changes too. Unless I'm missing a reason we need it.

Reply:
If we decide to go for a subkind we should remove the enforced origin label; as we're already going to use a subkind for the OpenSSH nodes rather than a label, I feel like it's cleaner to use a subkind for AppServer rather than a label. Either would work, really, but it's something we have to decide before release and then we kinda have to stick to the decision.